ide/diff/src/org/netbeans/modules/diff/builtin/visualizer/editable/EditableDiffView.java

```diff
@@ -122,13 +124,18 @@ public class EditableDiffView extends DiffControllerImpl implements DiffView, Do
 
     final JLabel fileLabel1 = new JLabel();
     final JLabel fileLabel2 = new JLabel();
+    final JLabel lineEndingLabel1 = new JLabel();
+    final JLabel lineEndingLabel2 = new JLabel();
     final JPanel filePanel1 = new JPanel();
     final JPanel filePanel2 = new JPanel();
     final JPanel textualPanel = new JPanel();
     final JTabbedPane jTabbedPane;
     final JComponent view;
     final JSplitPane jSplitPane1 = new JSplitPane();
 
+    private String lineEnding1;
+    private String lineEnding2;
+
```

Comment:
These are not required. `lineEnding1` and `lineEnding2` are only required in `RefreshDiffTask#run` and thus can be declared there.